src/main/java/core/basesyntax/QuadrangularArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface QuadrangularArea {
```

Comment:
Don't use verbs for class/interface names. It's better to name an interface with a noun or an adjective. For example, `Quadrangular` or `QuadrangularShape` might be more appropriate names for this interface.